members.html

```diff
@@ -393,6 +393,94 @@ <h2>Director of Professional Development</h2>
         </div>
     </section>
 
+    <!-- =======  Directors Section ======= -->
+
+    <section id="eboard" class="eboard">
```

Comment:
There's a lot of whitespace between the eboard section and the directors section. Could you remove lines 394 and 398? This will make the eboard section and directors section into one larger section and remove the whitespace.